src/WebJobs.Script/Diagnostics/OpenTelemetry/OpenTelemetryConfigurationExtensions.cs

```diff
@@ -87,7 +87,11 @@ private static IOpenTelemetryBuilder ConfigureTracing(this IOpenTelemetryBuilder
         {
             return builder.WithTracing(builder =>
             {
-                builder.AddSource("Azure.*")
+                builder
+                .AddSource("Azure.*")
+                .AddSource("Microsoft.Azure.Webjobs.Extensions.*")
+                .AddSource("WebJobs.Extensions.DurableTask")
```

Comment:
@bachuv -- would it be possible to change durables webjob source to `Microsoft.Azure.WebJobs.Extensions.DurableTask`?

Reply:
We released the Durable WebJobs extension package with this source name last week. Is it fine to change the name now since it's already released?

Reply:
Ah I suppose not if it is already released.